example/mavshell/mavshell.cpp

```diff
@@ -0,0 +1,107 @@
+#include <future>
+#include <mavsdk/mavsdk.h>
+#include <mavsdk/plugins/shell/shell.h>
+#include <iostream>
+
+using namespace mavsdk;
+
+#define ERROR_CONSOLE_TEXT "\033[31m" // Turn text on console red
+#define TELEMETRY_CONSOLE_TEXT "\033[34m" // Turn text on console blue
+#define NORMAL_CONSOLE_TEXT "\033[0m" // Restore normal console colour
+
+bool are_arguments_valid(const int, char**);
+void print_usage(const std::string& bin_name);
+bool start_discovery(Mavsdk&, const std::string&);
+void wait_until_discover(Mavsdk&);
+void run_interactive_shell(Mavsdk&, const uint16_t);
+
+int main(int argc, char** argv)
+{
+    if (!are_arguments_valid(argc, argv)) {
+        const auto binary_name = argv[0];
+        print_usage(binary_name);
+        return 1;
+    }
+
+    Mavsdk dc;
```

Comment:
Haha, just keep copying `dc`. It's not `DroneCore` anymore.

```suggestion
    Mavsdk mavsdk;
```